src/sentry/dynamic_sampling/rules/base.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, OrderedDict, Set
+
+import sentry_sdk
+
+from sentry import quotas
+from sentry.dynamic_sampling.rules.biases.base import Bias, BiasParams
+from sentry.dynamic_sampling.rules.combine import get_relay_biases_combinator
+from sentry.dynamic_sampling.rules.logging import log_rules
+from sentry.dynamic_sampling.rules.utils import BaseRule, RuleType, get_enabled_user_biases
+from sentry.models import Project
+
+ALWAYS_ALLOWED_RULE_TYPES = {RuleType.UNIFORM_RULE}
+
+
+def _get_guarded_blended_sample_rate(project: Project) -> float:
+    sample_rate = quotas.get_blended_sample_rate(project)
+
+    if sample_rate is None:
+        raise Exception("The method get_blended_sample_rate() returned None")
```

Comment:
```suggestion
        raise Exception("The method get_blended_sample_rate() returns none")
```
or even better to stick with old message, so sentry can group it i guess.
```
get_blended_sample_rate returns none
```